boards/imix/src/main.rs

```diff
@@ -133,6 +147,17 @@ static mut RF233_BUF: [u8; radio::MAX_BUF_SIZE] = [0x00; radio::MAX_BUF_SIZE];
 static mut RF233_REG_WRITE: [u8; 2] = [0x00; 2];
 static mut RF233_REG_READ: [u8; 2] = [0x00; 2];
 
+// Same as above ^^ for the UDP syscall interface
+const UDP_HDR_SIZE: usize = 8;
+const PAYLOAD_LEN: usize = 200;
+const DEFAULT_CTX_PREFIX_LEN: u8 = 8;
+static DEFAULT_CTX_PREFIX: [u8; 16] = [0x0 as u8; 16];
```

Comment:
Needless type-cast and but if you want to be more explicit I think it is more idiomatic to write ```static DEFAULT_CTX_PREFIX: [u8; 16] = [0x0_u8; 16];``` instead!

Also, why `immutable static` instead of `const` here I believe these are equivalent?! (Doesn´t matter to me but would be preferable to be consistent)